backend/FwLite/LcmCrdt/CrdtProjectsService.cs

```diff
@@ -124,13 +122,19 @@ public async Task<CrdtProject> CreateProject(CreateProjectRequest request)
                 await SeedSystemData(serviceScope.ServiceProvider.GetRequiredService<DataModel>(), projectData.ClientId);
             await (request.AfterCreate?.Invoke(serviceScope.ServiceProvider, crdtProject) ?? Task.CompletedTask);
         }
-        catch(Exception e)
+        catch (Exception e)
         {
             logger.LogError(e, "Failed to create project {Project}, deleting database", crdtProject.Name);
             activity?.AddException(e);
             await db.Database.CloseConnectionAsync();
-            EnsureDeleteProject(sqliteFile);
-            throw;
```

Comment:
We lost this throw, I don't think we want to lose that.

Reply:
Oh, dang, right 👍